src/components/ShoppingCart.jsx

```diff
@@ -14,24 +14,24 @@ const CartButton = styled(Button)`
   height: 3rem;
   width: 3rem;
 `
-// #endregion
+//#endregion
 
 const ShoppingCart = () => {
-  const [open, setOpen] = useState(false);
-  const [origin, setOrigin] = useState({});
+  const [open, setOpen] = useState(false)
+  const [origin, setOrigin] = useState({})
 
   const openModal = e => {
-    const rect = e.currentTarget.getBoundingClientRect();
-    setOrigin({ left: rect.left - rect.width * 0.5, top: rect.top - rect.height * 0.5 });
-    setOpen(true);
+    const rect = e.currentTarget.getBoundingClientRect()
+    setOrigin({ left: rect.left - rect.width * 0.5, top: rect.top - rect.height * 0.5 })
```

Comment:
Placement of the modal on the screen?

Reply:
I get the bounding center coordinates of the clicked object, in this case the pizza button.

Then, when the modal gets instantiated, it starts from these coordinates and animates out into the center of the screen.
So we get a very subtle "pop out from the button"-effect :)

Reply:
A nice! I liked the UI, simple and clean!